lib/chef/resource/aws_rds_instance.rb

```diff
@@ -0,0 +1,33 @@
+require 'chef/provisioning/aws_driver/aws_resource'
+
+
+class Chef::Resource::AwsRdsInstance < Chef::Provisioning::AWSDriver::AWSResource
+  aws_sdk_type AWS::RDS::DBInstance
+
+  attribute :db_instance_identifier, kind_of: String, name_attribute: true
+
+  attribute :engine, kind_of: String
+  attribute :engine_version, kind_of: String
+  attribute :db_instance_class, kind_of: String
+  attribute :multi_az, default: false, kind_of: [TrueClass, FalseClass]
+  attribute :allocated_storage, kind_of: Integer, default: 5
+  attribute :iops, kind_of: Integer
+  attribute :publicly_accessible, kind_of: [TrueClass, FalseClass], default: false
+  attribute :master_username, kind_of: String
+  attribute :master_user_password, kind_of: String
+  attribute :db_name, kind_of: String
+  attribute :db_port, kind_of: Integer
+
```

Comment:
The RDS API supports tagging on creation, so we can thread this through to the creation call pretty easily.